docs/08-how-to-submit-a-proof/01-javascript-example.md

```diff
@@ -0,0 +1,179 @@
+---
+title: JavaScript Example
+---
+
+## How to Submit a Proof
+
+Follow the instructions below to submit a proof using JavaScript. This example demonstrates submission of different proof types and their events.
+
+## Prerequisites
+
+Before you begin, ensure you have Node.js installed on your machine. You can download and install it from the [official Node.js website](https://nodejs.org/).
+
+### Installing Node.js
+
+1. **Download Node.js**:
+   - Visit the [Node.js download page](https://nodejs.org/).
+   - Download the LTS (Long Term Support) version for your operating system.
+
+2. **Install Node.js**:
+   - Run the downloaded installer and follow the instructions.
+   - Ensure that the installer includes `npm` (Node Package Manager).
+
+3. **Verify Installation**:
+   - Open a terminal or command prompt.
+   - Run the following commands to verify that Node.js and npm are installed correctly:
+     ```shell
+     node -v
+     npm -v
+     ```
+   - You should see the version numbers of Node.js and npm.
+
+Now you are ready to follow the instructions below to submit a proof.
+
+### Initialize Your Project
+
+1. Create a new directory for your project, navigate into it:
+
+    ```shell
+    mkdir my-zkverify-project
+    cd my-zkverify-project
+    ```
+2. Initialize a new npm project:
+
+    ```shell
+    npm init -y
+    ```
+3. Install the required packages:
+
+    ```shell
+    npm install @polkadot/api @polkadot/keyring dotenv
+    ```
+4. Create a `.env` file:
+
+    ```shell
+    echo -e "WEBSOCKET=wss://testnet-rpc.zkverify.io\nPRIVATE_KEY=\"below giant big little small royal three two one yes test tag\"" > .env
+    ```
+5. Update the `.env` file values:
+
+   - WEBSOCKET: The zkVerify Websocket url
+   - PRIVATE_KEY: Your testnet seedphrase enclosed in quotation marks
+
+### Example Script
+
+Create a file named `submitProof.js` and paste the following script into it:
+
+```javascript
+require('dotenv').config();
+const { ApiPromise, WsProvider } = require('@polkadot/api');
+const { Keyring } = require('@polkadot/keyring');
+
+const proofs = {
+   fflonk: {
+      pallet: 'settlementFFlonkPallet'
+   },
+   boojum: {
+      pallet: 'settlementZksyncPallet'
+   }
+};
+
+if (!process.env.WEBSOCKET || !process.env.PRIVATE_KEY) {
+   throw new Error('Required environment variables WEBSOCKET or PRIVATE_KEY are not set.');
+}
+
+const proofType = process.argv[2];
+const proofHash = process.argv[3];
```

Comment:
This is not 'proofHash' but simply 'proof'. Please modify in the whole code (remember comments and error messages as well)